CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 All notable changes to the "vscode-ansible" extension will be documented in this file.
 
+## 0.3.1
+#### Bug fixing
+- Fix bug in ssh flow when no workspace.
```

Comment:
Fixed bug

Reply:
and maybe "when no active workspace"?

Reply:
word updated. it is not active workspace, but no vscode workspace, eg. user directly open a file.